editoast/src/core/path_properties.rs

```diff
@@ -78,6 +78,8 @@ pub struct OperationalPointOnPath {
     extensions: OperationalPointExtensions,
     /// Distance from the beginning of the path in mm
     position: u64,
+    /// Importance of the operational point
+    weight: Option<u8>
```

Comment:
```suggestion
    /// Importance of the operational point
    #[schema(required)]
    weight: Option<u8>
```

will mark the field as required (we serialize `None` to `null` by default), so the field will never be `undefined` in the frontend

Reply:
Done 👍🏽